walk-in-interview/src/main/webapp/account/log-in/business/script.js

```diff
@@ -4,30 +4,59 @@
  */
 
 // TODO(issue/21): get the language from the browser
-const CurrentLocale = 'en';
+const CURRENT_LOCALE = 'en';
 
 /**
  * Import statements are static so its parameters cannot be dynamic.
  * TODO(issue/22): figure out how to use dynamic imports
  */
-import {AppStrings} from '../../strings.en.js';
-import {Auth} from '../firebase-auth.js';
-import {setErrorMessage} from '../../common-functions.js';
+import {AppStrings} from '../../../strings.en.js';
+import {Auth} from '../../firebase-auth.js';
+import {USER_TYPE_COOKIE_PARAM, USER_TYPE_BUSINESS,
+  setCookie, setErrorMessage} from '../../../common-functions.js';
 
+const AUTH_STRINGS = AppStrings['auth'];
 const COMMONG_STRINGS = AppStrings['log-in'];
 const STRINGS = AppStrings['business-log-in'];
 const LOGIN_HOMEPAGE_PATH = '../index.html';
-const HOMEPAGE_PATH = '../../index.html';
+const HOMEPAGE_PATH = '../../../index.html';
 
 const INVALID_EMAIL_ERROR_CODE = 'auth/invalid-email';
 const USER_DISABLED_ERROR_CODE = 'auth/user-disabled';
 const USER_NOT_FOUND_ERROR_CODE = 'auth/user-not-found';
 const WRONG_PASSWORD_ERROR_CODE = 'auth/wrong-password';
 
 window.onload = () => {
+  Auth.subscribeToUserAuthenticationChanges(onLogIn, onLogOut, onDefault);
   renderPageElements();
 };
 
+// TODO(issue/101): Display button according to log in status;
+// i.e. implement onLogIn, onLogOut, onDefault
+
+/**
+ * What to do after the user signed in and the session cookie is created.
+ */
+function onLogIn() {
+  // TODO(issue/100): set the cookie at the server side instead
+  setCookie(USER_TYPE_COOKIE_PARAM, USER_TYPE_BUSINESS);
```

Comment:
how do you know that the user is a business user. can a user navigate to your business page while being logged in as an applicant? (i'm pretty sure they can since i don't see any restriction anywhere)

Reply:
as discussed offline, the issue is because the user type is controlled by client not server (as we discussed before) so cannot have the server reject a page load if the user type doesn't match the expectation